funcx_endpoint/funcx_endpoint/endpoint/interchange.py

```diff
@@ -156,7 +156,9 @@ def load_config(self):
                 executor.endpoint_id = self.endpoint_id
             else:
                 if not executor.endpoint_id == self.endpoint_id:
-                    raise Exception("InconsistentEndpointId")
+                    eep_id = str(executor.endpoint_id)
+                    sep_id = str(self.endpoint_id)
+                    raise Exception(f"InconsistentEndpointId: {eep_id} != {sep_id}")
```

Comment:
Since you are clarifying anyway, how about a bit more?  
                    raise Exception(f"InconsistentEndpointId: executor({eep_id}) != {sep_id}")

Reply:
Good idea.  Just pushed a change.  Per out-of-band discussion, should address it?